package.json

```diff
@@ -80,7 +80,7 @@
     "can-dom-data": "1.0.2",
     "can-dom-data-state": "1.1.1",
     "can-dom-events": "1.3.11",
-    "can-dom-mutate": "1.3.7",
+    "can-dom-mutate": "2.0.2",
```

Comment:
A major version bump would mean a major version bump for CanJS, right? I don’t think we should change that yet.

Reply:
Fixed!